help/C/Help_ch_Transactions.xml

```diff
@@ -899,180 +905,543 @@ Author:
             sub menu to initiate importing transactions.
       </para>      
     </sect2>
+    
     <sect2 id="trans-import-qif">
       <title>Import QIF</title>
       <para>To import data from Quicken®, MS Money, or other programs that use QIF(Quicken® Interchange Format),
             you must first export your data to a QIF file. One way to do this is to export each account as a 
             separate QIF file. An easier way, available in Quicken® 98 and beyond, is to export all accounts at
             once into a single QIF file. Check your program's manual to determine if this option is available.
       </para>
-      <sect3><title>To import QIF files:</title><para> </para></sect3>
       <sect3>
-        <title>Load all of the QIF files containing data you wish to import</title>
-        <para>To do this, select
+        <title>To import QIF files:</title>
+        <para>Select
           <menuchoice>
             <guimenu>File</guimenu>
             <guisubmenu>Import</guisubmenu>
-            <guimenuitem>Import QIF...</guimenuitem>
+            <guimenuitem>Import QIF</guimenuitem>
```

Comment:
It should be the other way: The error is in the program. This menu entries should show a hellipsis, because a dialog follows in contrast to a command, which is immediately executed like File->Save.